src/core/data.jl

```diff
@@ -41,11 +27,12 @@ end
 
 ""
 function _get_repairable_items(network::Dict{String,Any})
-    repairs = Array{Tuple{String,String},1}()
+    repairs = Dict{String, Array{String}}()
```

Comment:
Here I think you want `Array{String,1}`, `Array{String}` is not prescriptive about the dimension of the Array.  BTW, you can also use `Vector{String}` and `Matrix{String}` as shorthands for `Array{String,1}` and `Array{String,2}` respectively.